slack/1.0/modules/ROOT/pages/index.adoc

```diff
@@ -0,0 +1,53 @@
+= Slack Connector 1.0 - Mule 4
+
+Support Category: https://www.mulesoft.com/legal/versioning-back-support-policy#anypoint-connectors[Select]
+
+Anypoint Connector for Slack (Slack Connector) enables access to the Slack platform. This connector exposes all of the operations provided by Slack's REST API.
+
+For compatibility information and fixed issues, see the Slack Connector Release Notes.
+
+== Prerequisites
+
+To use this connector, you must be familiar with:
+
+* The Slack API
+* Anypoint Connectors
+* Mule runtime engine (Mule)
+* Elements and global elements in a Mule flow
+* Creating a Mule app using Anypoint Studio (Studio)
+
+Before creating an app, you must have:
+
+* Credentials to access the Slack target resource
+* Anypoint Studio version 7.1 or later
+
+== Common Use Cases for the Connector
+
+*Need to add some use cases here*
+
+For examples, see xref:slack-connector-examples.adoc[Examples].
+
+== Audience
+
+* New users, read:
+** xref:slack-connector-studio.adoc[Use Studio to Configure the Connector] to create your Mule app
+** xref:slack-connector-examples.adoc[Examples], which provide one or more use cases for the connector
+* Advanced users, read:
+** xref:slack-connector-xml-maven.adoc[XML and Maven Support]
+** xref:slack-connector-examples.adoc[Examples], which provides one or more use cases for the connector
+
+== Authentication Types
+
+Slack Connector uses the OAuth 2.0 authentication type, which uses access tokens provided by the authorization server to get access to protected data.
```

Comment:
should we have a section where it is explained how to get the token initially? like this one on Salesforce? https://docs.mulesoft.com/salesforce-connector/10.3/salesforce-connector-studio#oauth-2-0